build-deb-package

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file BOOST_LICENSE_1.0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+set -xeu
+
+# if the requested build identifier matches a git tag, use that directly,
+# otherwise use git revision description (note, we implicitly assume that all
+# tags and revision descriptions will begin with `v`)
+versiontag=$(git describe)
```

Comment:
Why did you stripped the version scheme conversion code from here (compared to https://github.com/sociomantic-tsunami/libmxnet-deb/pull/2/files#diff-b0da275520918e23dd615e2a747528f1R28)? We need to convert the version from SemVer to Debian scheme. You could get the code from [MakD](https://github.com/sociomantic-tsunami/makd/blob/v1.x.x/Makd.mak#L251-L256).

To void duplication, I wonder if it's not worth adding MakD as a submodule just for the packaging building...